### ISSUE TITLE
Mace does not support onnx op type Tdnn yet

### ISSUE THREAD

**Issue by cocoakeith** [XiaoMi/kaldi-onnx#31]
Tdnn这块暂时没支持么？
![image](https://user-images.githubusercontent.com/12630218/92934448-1264ba00-f47a-11ea-9def-6ef69d4eeff3.png)

**Comment by lu229** [comment]
@cocoakeith 是的, 还没有支持, 之前看到一个开发者使用了一个变通的方法: 将kaldi里的模型定义文件（final.config）里对Tdnn的定义转成等价的Affine，然后kaldi拿转换好的定义文件去训练网络. (目前Affine是支持的)

**Comment by beyondych11** [comment]
@lu229 Hi！mace这块只是针对于kaldi的am，那怎样将decode部分也部署到移动端呢？大佬有这样做过或有什么建议吗？
